core/tools/mavlink_camera_manager/bootstrap.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
i would call this an install.sh rather than bootstrap

Reply:
I agree, sadly the name is also used by mavlink-routerd, I'll send a PR to fix both layer for consistency.